app/views/discovered_hosts/_discovered_host.html.erb

```diff
@@ -1,6 +1,6 @@
 <td class="ellipsis"><%= link_to (discovery_status_icon(host) + '&nbsp;'.html_safe + trunc_with_tooltip(h(host.name))), discovered_host_path(host) %></td>
 <td class="hidden-tablet hidden-xs"><%= host.try(:hardware_model_name) || 'N/A' %></td>
-<td class="hidden-tablet hidden-xs"><%= host.ip %></td>
+<td class="hidden-tablet hidden-xs"><%= discovered_ip(host) %></td>
```

Comment:
Wouldn't it be simpler to add a column for the IPv6 field, similar to how it's done on the Hosts index page? This would also help maintain consistency between the two pages.